src/style/shared/colors.less

```diff
@@ -95,6 +95,14 @@
 @search-text: #8e8e8e;
 @search-bright-text: #cacaca;
 
+/* Libraries */
+@libraries-dialog-dark-gray: #282828;
```

Comment:
Should these colors be defined symbolically? Should we be using color variables from somewhere else to match our UI color?